probeservices/checkin_test.go

```diff
@@ -0,0 +1,66 @@
+package probeservices_test
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/ooni/probe-engine/model"
+)
+
+func TestCheckInSuccess(t *testing.T) {
+	client := newclient()
+	client.BaseURL = "https://ams-pg-test.ooni.org"
+	config := model.CheckInConfig{
+		Charging:        true,
+		OnWiFi:          true,
+		Platform:        "string",
+		ProbeASN:        "string",
+		ProbeCC:         "string",
+		RunType:         "string",
+		SoftwareVersion: "string",
```

Comment:
Can we use realistic values here?

Reply:
Platform:        "android",
		ProbeASN:        "AS12353",
		ProbeCC:         "PT",
		RunType:         "?",
		SoftwareVersion: "2.7.1",

Reply:
I don't know what runtype is. @FedericoCeratto ?

Reply:
solved